controllers/admin/AdminOrdersController.php

```diff
@@ -2677,6 +2678,7 @@ public function ajaxProcessEditProductOnOrder()
             'documents_html' => $this->createTemplate('_documents.tpl')->fetch(),
             'shipping_html' => $this->createTemplate('_shipping.tpl')->fetch(),
             'customized_product' => is_array(Tools::getValue('product_quantity')),
+            'displayExcludedTaxMethod' => (bool)Group::getPriceDisplayMethod($this->context->customer->id_default_group),
```

Comment:
That's a visual detail but for visibility we usually put a space between (bool) and the element being casted.